packages/experimental/Shimmer/src/ShimmerTokens.ts

```diff
@@ -0,0 +1,15 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { ShimmerTokens } from './Shimmer.types';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { getCurrentAppearance } from '@fluentui-react-native/theming-utils';
+
+export const defaultShimmerTokens: TokenSettings<ShimmerTokens, Theme> = (theme: Theme) =>
+  ({
+    angle: 0,
+    delay: 0,
```

Comment:
This value is different on win32, please add a .win32 version of this file.

Reply:
Prior to this PR, the Shimmer.styling.ts (and Shimmer.styling.win32.ts) had the token values directly in the file itself. Separate ShimmerTokens.ts files were _not_ present.

I have separated only the Shimmer.styling.ts 's tokens into the ShimmerTokens.ts file and not the win32 fork. So, the ShimmerTokens.ts file is not referenced / being used for win32. Win32 specific values still reside in the Shimmer.styling.win32.ts file.

Reply:
I don't think it makes sense for defaultShimmerTokens for the win32 platform to be in the styling file in the first place actually. Can we move the defaultShimmerTokens from the Shimmer.styling.win32 to ShimmerTokens.win32 file?

Reply:
Acceptable. Have updated win32 Shimmer as well. Moved tokens out into separate ShimmerTokens.win32 file.